jmeter/0.2.0/README.md

```diff
@@ -0,0 +1,159 @@
+
+# Install extra packages
+# See https://github.com/gliderlabs/docker-alpine/issues/136#issuecomment-272703023
+# Change TimeZone TODO: TZ still is not set!
+ARG TZ="Europe/Amsterdam"
+RUN    apk update \
+	&& apk upgrade \
+	&& apk add ca-certificates libc6-compat \
+	&& update-ca-certificates \
+	&& apk add --update openjdk8-jre tzdata curl unzip bash \
+	&& apk add --no-cache nss \
+	&& rm -rf /var/cache/apk/*
+
+# install jmeter
+RUN mkdir -p /tmp/dependencies  \
+	&& curl -L --silent ${JMETER_DOWNLOAD_URL} >  /tmp/dependencies/apache-jmeter-${JMETER_VERSION}.tgz  \
+	&& mkdir -p /opt  \
+	&& tar -xzf /tmp/dependencies/apache-jmeter-${JMETER_VERSION}.tgz -C /opt  \
+	&& rm -rf /tmp/dependencies
+
+# Set global PATH such that "jmeter" command is found
+ENV PATH $PATH:$JMETER_BIN
+
+# Entrypoint has same signature as "jmeter" command
+COPY entrypoint.sh /
+
+WORKDIR	/keptn/jmeter
+
+RUN chmod +x /entrypoint.sh
+
+ENTRYPOINT ["/entrypoint.sh"]
+```
+
+The `entrypoint.sh` file executes JMeter using the passed arguments. You can also uncomment the commands below to enable some debugging steps.
+
+```bash
+#!/bin/bash
+echo "jmeter args=$@"
+
+echo "START Running Jmeter on `date`"
+
+jmeter $@
+
+echo "END Running Jmeter on `date`"
+
+# Debugging
+# echo "Test.log"
+# cat /keptn/jmeter/test.log
+
+# echo "log.tlf"
+# cat /keptn/jmeter/log.tlf
+```
+
+Now you will need to build the Docker image and push it to your image registry.
+
+```bash
+docker build -t yourorg/jmeter:latest .
+docker push yourorg/jmeter:latest
+```
+
+## Configure JMeter using the Job-Executor
+
+### Step 1: Add the Job-Executor configuration file
+
+The following configuration will allow you to run the `basiccheck.jmx` (e.g., from the [Keptn Carts example](https://github.com/keptn/examples/tree/master/onboarding-carts/jmeter)) using the Job-Executor-Service whenever a test event is triggered.
+
+```yaml
+apiVersion: v2
+actions:
+  - name: "Run JMeter"
+    events:
+      - name: "sh.keptn.event.test.triggered"
+        jsonpath:
+          property: "$.data.test.teststrategy"
+          match: "performance"
+    tasks:
+      - name: "Run jmeter smoke tests"
+        files:
+          - jmeter/load.jmx
+        image: "docker.io/yourorg/jmeter:latest"
+        args:
+          - '-n'
+          - '-t'
+          - '/keptn/jmeter/load.jmx'
+          - '-JPROTOCOL=http'
+          - '-JSERVER_PROTOCOL=http'
+          - '-JVUCount=10'
+          - '-JLoopCount=10'
+          - '-JSERVER_URL=$(KEPTN_SERVICE).$(KEPTN_PROJECT)-$(KEPTN_STAGE).svc.cluster.local'
+          - '-j'
+          - '/keptn/jmeter/test.log'
+          - '-l'
+          - '/keptn/jmeter/log.tlf'
+```
+
+Add the job-executor configuration file to your Keptn service:
+
+```bash
+keptn add-resource --project=sockshop --service=carts --stage=staging --resource=jmeter.yaml --resourceUri=job/config.yaml
```

Comment:
we never explicitly mentioned that the file above is called `jmeter.yaml`, can you just add a quick info before the code above?